code/game/machinery/computer/_computer.dm

```diff
@@ -42,12 +42,11 @@
 	. += icon_keyboard
 
 	// This whole block lets screens ignore lighting and be visible even in the darkest room
-	// We can't do this for many things that emit light unfortunately because it layers over things that would be on top of it
 	var/overlay_state = icon_screen
 	if(machine_stat & BROKEN)
 		overlay_state = "[icon_state]_broken"
 	SSvis_overlays.add_vis_overlay(src, icon, overlay_state, layer, plane, dir)
```

Comment:
You can remove this one, it was only here as the backdrop to the half opacity one